example-doc-generator/python/crop_screenshots.py

```
@@ -0,0 +1,148 @@
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+crop_screenshots.py — Post-process pipeline screenshots by cropping UI chrome.
+
+Default margins are tuned for a 1720×968 headless Playwright / code-server viewport.
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)